skimage/measure/tests/test_fit.py

```diff
@@ -229,7 +229,22 @@ def test_ellipse_model_estimate_failers():
     # estimate parameters of real data
     model = EllipseModel()
     assert not model.estimate(np.ones((5, 2)))
+    # Before PR https://github.com/scikit-image/scikit-image/pull/3103
+    # This next line would return True on 32bit
     assert not model.estimate(np.array([[50, 80], [51, 81], [52, 80]]))
+    # This is the same test that causes the model to
+    # return true on 64 bit prooving that this was also an issue on
+    # 64 bit arch
+    assert not model.estimate(np.array([[50, 80], [51, 81], [52, 80.00000000001]]))
+    # Unfortunately, it seems to depend on the version of python that is used
+    # Therefore, test for many cases
```

Comment:
@hmaarrfk based on your comment, shouldn't this test be:

```python
assert not all(model.estimate(np.array([[50, 80], [51, 81], [52, 80 + 10**(-n)]]))
               for n in range(4, 12))
```
?

Reply:
@jni, refactored as per your suggestion. problem remains :/

Reply:
Which problem? Do they all pass for some Python versions?

Reply:
Well my **fix** causes the doctest to fail. But the doctest seems to make sense. https://travis-ci.org/scikit-image/scikit-image/jobs/432304643#L2823

I'm glad to make the test xfail for code self-documentation.

Reply:
OOOOoooooOOOOhhhh. Very interesting. Hmm, I think ellipse-fitting to a line should work, actually. =\